index.html

```diff
@@ -0,0 +1,124 @@
+        <input
+          id="number"
+          value="0"
+          type="number"
+          min="0"
+          max="100"
+          placeholder="Number"
+          name ="info"
+          required
+        />
+
+        <select name="info" id="dropdown" required class="continents">
+          <option value="">--Select your continent--</option>
+          <option value="Africa">Africa</option>
+          <option value="North America">North America</option>
+          <option value="South America">South America</option>
+          <option value="Europe">Europe</option>
+          <option value="Asia">Asia</option>
+          <option value="Australia">Australia</option>
+          <option value="Antarctica">Antarctica</option>
+        </select>
+
+        <p>Select your favourite code editor:</p>
+        <div>
+          <input type="radio" id="emacs" name="fav_language" value="Emacs" class="radio-buttons"/>
+          <label for="html">Emacs</label>
+        </div>
+
+        <div>
+          <input type="radio" id="vim" name="fav_language" value="Vim" class="radio-buttons"/>
+          <label for="html">Vim</label>
+        </div>
+
+        <div>
+          <input type="radio" id="atom" name="fav_language" value="Atom" class="radio-buttons"/>
+          <label for="html">Atom</label>
+        </div>
+
+        <div>
+          <input type="radio" id="vscode" name="fav_language" value="VSCode" class="radio-buttons"/>
+          <label for="html">VSCode</label>
+        </div>
+        <div>
+          <input
+            type="radio"
+            id="sublime text"
+            name="fav_language"
+            value="sublime text"
+            class="radio-buttons"/>
+          <label for="html">Sublime</label>
+        </div>
+        <!--Second Selection radio button-->
+        <p>Select your favourite genere of music:</p>
+        <div>
+          <input type="checkbox" id="rock" name="fav_language" value="Rock" class="check_boxes"/>
+          <label for="html">Rock</label>
+        </div>
+        <div>
+          <input
+            type="checkbox"
+            id="hip-hop"
+            name="fav_language"
+            value="hip-hop"
+            class="check_boxes"
+          />
+          <label for="html">Hip-Hop</label>
+        </div>
+        <div>
+          <input
+            type="checkbox"
+            id="electronic"
+            name="fav_language"
+            value="electronic"
+            class="check_boxes"
+          />
+          <label for="html">Electronic</label>
+        </div>
+        <div>
+          <input
+            type="checkbox"
+            id="classical"
+            name="fav_language"
+            value="classical"
+            class="check_boxes"
+          />
+          <label for="html">Classical</label>
+        </div>
+        <div>
+          <input type="checkbox" id="pop" value="pop" class="check_boxes"/>
+          <label for="html">Pop</label>
+        </div>
```

Comment:
Also Adding proper `<li>` tag to list all of the possible favourite music genere input is good practice for proper HTML tag semantics. you can commit changes with my suggesion below, or make the changes yourself following the proper rules of using a `<li>` tag 
```suggestion
        <ul>
          <li>
            <input type="checkbox" id="rock" name="fav_language" value="Rock" class="check_boxes"/>
            <label for="rock">Rock</label>
          </li>
          <li>
              <input
              id="hip-hop"
              type="checkbox"
              name="fav_language"
              value="hip-hop"
              class="check_boxes"
            />
            <label for="hip-hop">Hip-Hop</label>
          </li>
          <li>
              <input
              id="electronic"
              type="checkbox"
              value="electronic"
              class="check_boxes"
            />
            <label for="electronic">Electronic</label>
          </li>
          <li>
              <input
              id="classical"
              type="checkbox"
              value="classical"
              class="check_boxes"
            />
          <label for="classical">Classical</label>
          </li>
          <li>
            <input id="pop" type="checkbox" value="pop" class="check_boxes"/>
            <label for="pop">Pop</label>
          </li>
        </ul>
```